topic-operator/src/main/java/io/strimzi/operator/topic/Session.java

```diff
@@ -267,6 +297,54 @@ public void handle(Long oldTimerId) {
             });
     }
 
+    private Properties adminClientProperties() {
+        Properties kafkaClientProps = new Properties();
+        kafkaClientProps.setProperty(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, config.get(Config.KAFKA_BOOTSTRAP_SERVERS));
+        kafkaClientProps.setProperty(StreamsConfig.APPLICATION_ID_CONFIG, config.get(Config.APPLICATION_ID));
+
+        String securityProtocol = config.get(Config.SECURITY_PROTOCOL);
+        boolean tlsEnabled = Boolean.parseBoolean(config.get(Config.TLS_ENABLED));
+
+        if (tlsEnabled && !securityProtocol.isEmpty()) {
+            if (!securityProtocol.equals("SASL_SSL") || !securityProtocol.equals("SSL")) {
```

Comment:
@tombentley @adinhodovic I think this should be `&&`? IMHO this is what failed all the system tests.

Reply:
Yep